client/components/LoginForm/LoginForm.js

```diff
@@ -45,30 +49,53 @@ class LoginForm extends Component {
     }
   }
 
-  onBlur (event) {
-    event.preventDefault()
+  onBlur(event) {
+    event.preventDefault();
```

Comment:
Remove semicolons, following the linting guidelines for standardjs, unless the semicolon is absolutely required